rules/windows/image_load/image_load_in_memory_powershell.yml

```
@@ -53,7 +53,11 @@ detection:
             - '\thor64.exe'
             - '\thor.exe'
        # User: 'NT AUTHORITY\SYSTEM'   # if set, matches all powershell processes not launched by SYSTEM
-    condition: selection and not 1 of filter*
+    filter_aurora:
+        # This filter is to avoid a race condition FP with this specific ETW provider in aurora
+        Provider_Name: Microsoft-Windows-Kernel-Process
+        Image: null
```

Can't we just look for `Image: null` here? 
Combining a `Field: null` with something else is always problematic for some backends. 
```suggestion
        Image: null
```

---

Yes technically any `Image: null` is a problem so we don't really care about the provider. So I think it's okay to remove it.